webapp/src/routes/home/game/index.js

```diff
@@ -17,12 +21,35 @@ export class Game {
     this.htmlElement.append(this._app.renderer.view);
 
     this.background = new Background({ width: this.width, height: this.height });
-    this.warehouse = new Warehouse({ rendererWidth: this.width, rendererHeight: this.height, scale: 1/3 });
+    this.warehouse = new Warehouse({ rendererWidth: this.width, rendererHeight: this.height, scale: 1 / 3 });
 
     this.stage.addChild(this.background.stage);
     this.stage.addChild(this.warehouse.stage);
+
+    this.displayFences().map(el => this.stage.addChild(el.stage));
   }
 
+  displayFences() {
+    const firstColumnXPos = 100;
+    const secondColumnXPos = this.width - 100;
+    const initialYPos = 250;
+    let increaseYValue = 0;
+    let fenceArray = [];
+
+    for (let i = 0; i < FENCES_ROWS; i++) {
+      for (let j = 0; j < FENCES_COLUMNS; j++) {
+        fenceArray.push(
+          new Fence({
+            scale: 1 / 3,
+            column: j,
+            position: { x: j ? secondColumnXPos : firstColumnXPos, y: initialYPos + increaseYValue },
+          }));
+      }
+      increaseYValue += (this.height - initialYPos) / FENCES_ROWS + 10;
```

Comment:
trzeba to trzymać w mutowalnej zmiennej? wydaje mi się, że da się to wyliczyć na podstawie iteracji w tablicy, w stylu:

`const yOffset = i * ((this.height - initialYPos) / FENCE_ROWS + 10)`

dodatkowo: czym jest to `10`? 😄

Reply:
10 to taki dodatkowy offset, bo z wyliczenia tego mimo wszystko są trochę zbyt blisko